configure

```diff
@@ -583,6 +618,12 @@ while [[ ! -z $1 ]] ; do
 #      HDF5LIB="-I$VALUE/include $VALUE/lib/libhdf5_hl.a $VALUE/lib/libhdf5.a -lm"
 #      echo "Using HDF5LIB: $HDF5LIB"
 #      ;;
+    "--with-readline")
```

Comment:
> Do you recall the reason for bundling readline in the first place? Surely it's one of the most widely deployed packages in any linux -- isn't it like a dependency of the shell?

@rmcgibbo So the `readline` that ships with Amber (for cpptraj, formerly for sleap/gleap as @swails mentioned) is actually a stripped-down version of readline edited by Wei Zhang (I think) with only the core functionality there. I briefly looked into linking in `readline` externally but it seemed like more work than it was worth at the time. I'm not opposed to changes, but using this `readline` has been the path of least resistance thus far :-)